### PR TITLE
Chore: Use consistent badge style

### PR DESCRIPTION
**Pull request by fisker**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/master/CONTRIBUTING.md).
- [ ] The team has reached consensus on the changes proposed in this pull request. If not, I understand that the evaluation process will begin with this pull request and won't be merged until the team has reached consensus.

#### What is the purpose of this pull request? (put an "X" next to item)

[ ] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-change-proposal.md))
[ ] Add autofixing to a rule
[ ] Add a CLI option
[ ] Add something to the core
[x] Other, please explain:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -276,3 +276,5 @@ The following companies, organizations, and individuals support ESLint's ongoing
 [npm-url]: https://www.npmjs.com/package/eslint
 [downloads-image]: https://img.shields.io/npm/dm/eslint.svg?style=flat-square
 [downloads-url]: https://www.npmjs.com/package/eslint
+[github-actions-image]: https://img.shields.io/github/workflow/status/eslint/eslint/CI?style=flat-square
+[github-actions-image]: https://github.com/eslint/eslint/actions
```
Comment (kaicataldo) [inline README.md]:
```suggestion
[github-actions-url]: https://github.com/eslint/eslint/actions
```

Comment (fisker) [reply README.md]: Thanks

**Comment by kaicataldo** [comment]
Thoughts on making the styles consistent? We now have a mix of the square and rounded corners badges.

**Comment by fisker** [comment]
Seems no flat square badge for `FOSSA` so change the other 2 badges to rounded ?

**Comment by kaicataldo** [comment]
Sounds good to me!

**Comment by fisker** [comment]
@kaicataldo Done. [README.md](https://github.com/fisker/eslint/blob/actions-badge/README.md)

**Pull request merged**